cmd/mattermost/commands/config.go

```diff
@@ -163,10 +195,15 @@ func printConfigValues(configMap map[string]interface{}, configSetting []string,
 	}
 }
 
-// printMap takes a reflect.Value and return a string, recursively if its a map with the given tab settings.
+// prettyPrint the map
+func prettyPrint(configMap map[string]interface{}) string {
```

Comment:
This code obviously already exists, but I'm surprised we're adopting a custom output format here instead of just dumping JSON ready for consumption by other applications. Then again, this is probably "just for humans", so the existing format is fine.

Reply:
I was told to return a string in this function and do all printing inside the Show command.